target/linux/ath79/dts/ar9331_tplink_tl-wr710n-v1.dts

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9331.dtsi"
+
+/ {
+	model = "TP-Link TL-WR710N v1";
+	compatible = "tplink,tl-wr710n-v1", "qca,ar9331";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &led_system;
+		led-failsafe = &led_system;
+		led-running = &led_system;
+		led-upgrade = &led_system;
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		led_system: system {
+			label = "tl-wr710n:green:system";
+			gpios = <&gpio 27 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	reg_usb_vbus: reg_usb_vbus {
```

Comment:
maybe reg_usb_vbus: regulator { ?

Reply:
Changed

Reply:
@chunkeey @marcin1j  USB power is broken for me. I don't know if the problem is DTS or the GPIO

Reply:
Nevermind, for some reason in my build config there is no kmod-usb-core or kmod-usb-chipidea2 .....